src/constants/constants.js

```diff
@@ -5,3 +5,7 @@ export const sex_options = [
   { value: 'male', label: 'Male' },
   { value: 'female', label: 'Female' },
 ];
+
+export const config = {
+  API_URL: 'http://target-mvd-api.herokuapp.com/api/v1'
```

Comment:
Don't you have two separate `devConstants` and `prodConstants` files? If you don't, don't change anything and call me after you fixed this PR. But this URL shouldn't be on the repo.